stylix/maintainers.nix

```diff
@@ -65,6 +65,12 @@
     github = "osipog";
     githubId = 87434959;
   };
+  sand4rt = {
+    email = "17591696+sand4rt@users.noreply.github.com";
```

Comment:
If you do not wish to publish your email, it might be simpler to not include this optional field:


```suggestion
```